events/attributevalue.go

```diff
@@ -18,6 +18,71 @@ type DynamoDBAttributeValue struct {
 	dataType DynamoDBDataType
 }
 
+// Replicating this from aws sdk to prevent dependency bloating / potential dependency import cycles
```

Comment:
Ideally this comment would still follow standard godoc pattern (ie, `An AttributeValues is ...`). This might also be a good place to explain where it's replicated from (the AWS SDK) and how to update it in the future.